tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/routes/DownloadAllRoutes.scala

```diff
@@ -156,24 +159,32 @@ class DownloadAllRoutes @Inject()(@Named(InjectionKeys.RFC_8621) val authenticat
     val id: String = request.param(emailIdParam)
     SMono.fromCallable(() => messageIdFactory.fromString(id: String))
       .onErrorResume(e => SMono.error(MessageNotFoundException(id, e)))
-      .flatMap(messageId => SFlux(messageIdManager.getMessagesReactive(ImmutableList.of(messageId), FetchGroup.HEADERS_WITH_ATTACHMENTS_METADATA, mailboxSession))
+      .flatMap(messageId => SFlux(messageIdManager.getMessagesReactive(ImmutableList.of(messageId), FetchGroup.FULL_CONTENT, mailboxSession))
```

Comment:
Why we need to change this?

We can still iterate the LoadedAttachment (which are badly named as they are `MessageAttachmentMetadata`) and simply load their content using the AttachmentId and the BlobResolvers?

Reply:
I thought the pitch was :  attachments in the S3 blobstore got deleted by mistake? What was deleted exactly then? Rather confused with the description of the ticket then